lib/Model.php

```diff
@@ -426,7 +426,7 @@ public function __set($name, $value)
 		foreach (static::$delegate as &$item)
 		{
 			if (($delegated_name = $this->is_delegated($name,$item)))
-				return $this->$item['to']->$delegated_name = $value;
+				return $this->{$item['to']}->$delegated_name = $value;
```

Comment:
Good point. I've pushed a commit to that effect but I see a similar fix has already been done upstream :)